src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/model/Location.kt

```diff
@@ -69,9 +70,11 @@ data class RoomSchedule(
   val endDayOfWeek: DayOfWeek,
 
   @Schema(description = "Start time of this slot (24 hr clock, HH:MI)", example = "10:00", required = true)
+  @JsonFormat(pattern = "HH:mm")
   val startTime: LocalTime,
 
   @Schema(description = "End time of this slot (24 hr clock, HH:MI)", example = "16:00", required = true)
+  @JsonFormat(pattern = "HH:mm")
```

Comment:
This stops the times from including seconds.  Relates to this comment.

![image](https://github.com/user-attachments/assets/190cc009-0668-4db6-afe8-36a277c2bf28)